src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```

Comment:
we should not write static methods, refer to checklist
```suggestion
    public Ball getRandomBall() {
```